### PR TITLE
content_size validation added

### PR DESCRIPTION
**Pull request by reckerswartz**
## content_size validation added for 
:less_than, 
:less_than_or_equal_to, 
:greater_than, 
:greater_than_or_equal_to, 
:between

*writing test pending!

### CODE REVIEW

File: lib/active_storage_validations.rb [hunk 1]
```diff
@@ -44,6 +44,50 @@ def content_type_valid?(file)
...
+          file_size >= options[:less_than_or_equal_to]
+        when options[:greater_than].present?
+          file_size < options[:greater_than]
+        when options[:greater_than_or_equal_to].present?
+          file_size <= options[:greater_than_or_equal_to]
+      end 
+    end
+
+    def human_size(size)
+      ActiveSupport::NumberHelper.number_to_human_size(size)
```
Comment (igorkasyanchuk) [inline lib/active_storage_validations.rb]: I think you can do something like

delegate :number_to_human_size, to: ActiveSupport::NumberHelper

and remove this method

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ class Project < ApplicationRecord
 
   validates :title, presence: true
 
-  validates :preview, attached: true
+  validates :preview, attached: true, content_size: { less_than: 100.megabytes , message: 'is not given between size' }
```
Comment (igorkasyanchuk) [inline README.md]: maybe rename content_size to "size" or "file_size"
since it's more about file

**Comment by igorkasyanchuk** [comment]
@reckerswartz very good start, please check few comments and yes, at least a simple tests are needed

File: README.md (hunk 2 above)
Comment (reckerswartz) [reply README.md]: @igorkasyanchuk request to change it to active_storage_size prevent conflict with other 3party gems. please suggest.

Comment (igorkasyanchuk) [reply README.md]: this is for "size" and "file_size"? who is using such validations? Maybe file_size is available?

Comment (igorkasyanchuk) [reply README.md]: @reckerswartz do you have any other comments? this PR looks like 95% ready, if you can make fixes I'll merge it.

So I see 2 things which I would like to change:
- name of validator (find a better name if possible)
- use number_to_human_size (see my other comment in PR)

Comment (reckerswartz) [reply README.md]: @igorkasyanchuk sorry for delay!
updated as per requested but still test are pending :smile:

**Comment by igorkasyanchuk** [comment]
merged, I'll make a new release soon

**Comment by igorkasyanchuk** [comment]
@reckerswartz version 0.5 just released. thanks

**Pull request merged**